i18n/nl/cosmic_settings.ftl

```diff
@@ -0,0 +1,807 @@
+    .unavailable = Niet beschikbaar
+    .unknown = Status onbekend
+    .unmanaged = Onbeheerd
+    .unplugged = Kabel losgekoppeld
+
+remove-connection-dialog = Verbindingsprofiel verwijderen?
+    .vpn-description =  Om dit netwerk in de toekomst te kunnen gebruiken, moet u opnieuw een wachtwoord invoeren.
+    .wired-description = U moet dit profiel opnieuw aanmaken om het in de toekomst te kunnen gebruiken.
+
+vpn = VPN
+    .connections = VPN-verbindingen
+    .error = Kon geen VPN-configuratie aanmaken
+    .remove = Verbindingsprofiel verwijderen
+    .select-file = Selecteer VPN-configuratiebestand
+
+vpn-error = VPN-fout
+    .config = Kon geen VPN-configuratie aanmaken
+    .connect = VPN-verbinding mislukt
+    .connection-editor =  Verbindingseditor mislukt
+    .connection-settings = Het ophalen van instellingen voor actieve verbindingen is mistlukt
+    .updating-state = Kan status van netwerkbeheerder niet bijwerken
+    .wireguard-config-path = Ongeldig bestandspad voor de WireGuard-configuratie
+    .wireguard-config-path-desc = Het gekozen bestand moet op een lokaal bestandssysteem staan.
+    .wireguard-device = Kon WireGuard-apparaat niet aanmaken
+    .with-password = Kon { $field ->
+        *[username] de VPN-gebruikersnaam
+        [password] het VPN-wachtwoord
+        [password-flags] de VPN-wachtwoordvlaggen
+    } niet met nmcli toevoegen
+
+wired = Bedraad
+    .adapter = Bedrade adapter { $id }
+    .connections = Bedrade verbindingen
+    .devices = Bedrade apparaten
+    .remove = Verbindingsprofielen verwijderen
+
+wifi = Wifi
+    .adapter = Wifi-adapter { $id }
+    .forget = Dit netwerk vergeten
+
+wireguard-dialog = WireGuard-apparaat toevoegen
+    .description = Kies een apparaatnaam voor de WireGuard-configuratie.
+
+## Networking: Online Accounts
+
+online-accounts = Online accounts
+    .desc = Accounts toevoegen, IMAP en SMTP, bedrijfslogins
+
+# Bluetooth
+
+confirm = Bevestigen
+
+bluetooth = Bluetooth
+    .desc = Bluetooth-apparaten bewerken
+    .status = Dit systeem is zichtbaar als { $aliases } zolang de Bluetooth-instellingen open staan.
+    .connected = Verbonden
+    .connecting = Verbinding maken…
+    .disconnecting = Verbinding verbreken…
+    .connect = Verbinden
+    .disconnect = Verbinding verbreken
+    .forget = Vergeten
+    .dbus-error = Er is een fout opgetreden tijdens de interactie met DBus: { $why }
+
+bluetooth-paired = Eerder verbonden apparaten
+    .connect = Verbinden
+    .battery = { $percentage }% batterij
+
+bluetooth-confirm-pin = Bluetooth-pincode bevestigen
+    .description = Controleer of de volgende pincode overeenkomt met de pincode die op { $device } wordt weergegeven
+
+bluetooth-available = Nabije bluetooth-apparaten
+
+bluetooth-adapters = Bluetooth-adapters
+
+## Desktop
+
+desktop = Bureaublad
+
+## Desktop: Wallpaper
+
+wallpaper = Schermachtergrond
+    .change = Wijzig alle afbeeldingen
+        <#-- Wijzig alle afbeeldingen: Change every image -->
```

Comment:
Source: Change image every.
Strange word order, I have changed the word order to Change every image.